tests/unit/syslog_monitor_test.py

```diff
@@ -13,8 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ------------------------------------------------------------------------
-#
-# author: Imron Alston <imron@scalyr.com>
+# author: scalyr-cloudtech@scalyr.com
```

Comment:
I would change this to `scalyr@sentinelone.com`

Reply:
Will do